README.md

```diff
@@ -86,6 +89,15 @@ $ git clone https://github.com/acushlakoncept/tic-tac-toe.git
 $ bin/main.rb
 ~~~
 
+Testing
+
+To test the code, run `rspec` from root of the folder using terminal.
+> Rspec is used for the test.
+
+~~~bash
+$ gem install rspec
```

Comment:
Installing rspec should come before running rspec. Please reorder this section.

Reply:
> Installing rspec should come before running rspec. Please reorder this section.

Thanks.
We've re-arranged it.